targets/xorg

```diff
@@ -207,35 +174,12 @@ if [ -n "$removevideodrivers" ]; then
     remove $removevideodrivers
 fi
 
-# If this is a system with framebuffer compression, we need SNA+tearfree
-xorgconf='/usr/share/X11/xorg.conf.d/20-crouton-intel-sna.conf'
-if [ -n "$intelhasfbc" -a -n "$intelfbcsupport" ]; then
-    mkdir -p "${xorgconf%/*}"
-    ln -sfT /etc/crouton/xorg-intel-sna.conf "$xorgconf"
-else
-    # In case this got moved to a different system, delete the config
-    rm -f "$xorgconf"
-fi
+# Remove old Intel SNA workaround
+rm -f '/usr/share/X11/xorg.conf.d/20-crouton-intel-sna.conf'
```

Comment:
Are there any replacements for this workaround? It seems like *bionic* needs this workaround on my system (CBPro) with gnome/kde to fix #3988 / #3933